File: Barone.Client/src/app/modules/shared/models/estilos.model.ts
import { Constructable } from "../../../core/models/constructable";
import { RangoModel } from "./rango.model";

export class EstilosModel  extends Constructable<Partial<EstilosModel>> {
    public IdEstilo:number;
    public idRango:number;
    public Nombre:string;
    public ingredientes:string;
    public rangoPrecio:RangoModel;
}

File: Barone.api/Models/EstilosModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Barone.api.Models
{
    public class EstilosModel
    {
        [Key]
        public int IdEstilo { set; get; }
        public string Nombre { set; get; }
        public long idRango { set; get; }
        public string ingredientes { set; get; }
        public rangosPreciosModel rangoPrecio { set; get; }
        
    }
}